validation/run_single_q0.py

```diff
@@ -0,0 +1,34 @@
+import ROOT
```

Comment:
Can we make this follow the normal guidelines of using `if __name__ == "__main__"` and defining imports at the top?

Reply:
Since we don't do this for `validation/run_single.py` just yet -- can we get both of those in a separate PR?

Reply:
Added in #1241.